src/screens/auth/login/Login.tsx

```diff
@@ -155,6 +160,26 @@ export default class Login extends BaseScreen<Props, State> {
     }
   }
 
+  private noTenantFoundMessage(){
+    Alert.alert(
+      I18n.t('authentication.noTenantFoundTitle'),
+      I18n.t('authentication.noTenantFoundMessage'),
+      [
+        {text: I18n.t('authentication.addTenantButton') , onPress:() => {this.goToTenants(true)}},
```

Comment:
Add space after all opened curly braces and before closed ones.
Example:
`{ this.goToTenants(true) }`